api/krusty/openapitests/openapicustomschema_test.go

```diff
@@ -0,0 +1,644 @@
+// Copyright 2019 The Kubernetes Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+// The openapi field tests are separated due to the global OpenAPI variable causing
+// race conditions. As a result, these tests can't be run in parallel with the other tests.
+package openapitests //nolint
```

Comment:
Linter complained that this should be called `openapitests_test`. If I change it to `openapitests_test`, it complains saying that package names shouldn't have an underscore